ctags/main/read.c

```diff
@@ -555,15 +554,15 @@ extern char *readLine (vString *const vLine, MIO *const mio)
 		do
 		{
 			char *const pLastChar = vStringValue (vLine) + vStringSize (vLine) -2;
-			MIOPos startOfLine;
+			long startOfLine;
 
-			mio_getpos (mio, &startOfLine);
+			startOfLine = mio_tell(fp);
```

Comment:
how comes this is different in UCTags?  But well, whatever, practically it's the same.